C_CPP: File Management/Handling (#3926)
girlscript/winter-of-contributing: C_CPP/File Management or Handling/file_management_handling.md

> In this video i have discussed in detailed the basics of file management/handling in C language, like why we need it,how  does it works and its importance and related operations used in it.

Reviewer: ```suggestion
In this video, the basics of file management/handling in C language have discussed in details , viz - why we need it, how it works and its importance and related operations used in it.
```

## Here's a video giving the information on file management/handling in C:

> Link: https://drive.google.com/file/d/1lnJMwXshgJqMWmCeJe801PDVRtRoO7Cb/view?usp=sharing

Reviewer: Convert to relative link on an embedded image file
(For embedding just copy an image & press Ctrl+v)
